File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/Events/SeatTypeUpdated.cs
using System;

namespace ConferenceManagement
{
    public class SeatTypeUpdated : SeatTypeEvent
    {
        public SeatTypeUpdated() { }
        public SeatTypeUpdated(Guid seatTypeId, SeatTypeInfo seatTypeInfo)
            : base(seatTypeId, seatTypeInfo) { }
    }
}

File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/Events/SeatTypeUpdated.java
package com.microsoft.conference.management.domain.Events;

import com.microsoft.conference.management.domain.Models.SeatTypeInfo;

public class SeatTypeUpdated extends SeatTypeEvent {
    public SeatTypeUpdated() {
    }

    public SeatTypeUpdated(String seatTypeId, SeatTypeInfo seatTypeInfo) {
        super(seatTypeId, seatTypeInfo);
    }
}
